grails-app/services/org/pih/warehouse/inventory/CycleCountService.groovy

```diff
@@ -399,4 +399,47 @@ class CycleCountService {
             transaction.addToTransactionEntries(transactionEntry)
         }
     }
+
+    CycleCountItemDto updateCycleCountItem(CycleCountUpdateItemCommand command) {
+        CycleCountItem cycleCountItem = command.cycleCountItem
+        cycleCountItem.properties = command.properties
+        cycleCountItem.countIndex = command.recount ? 1 : 0
+        cycleCountItem.status = command.recount ? CycleCountItemStatus.INVESTIGATING : CycleCountItemStatus.COUNTING
+        cycleCountItem.dateCounted = new Date()
```

Comment:
I'm not sure if we want to auto-update the dateCounted here or if we just want to allow the user to (optionally) manually update it. Automatically makes sense to me but I think dateCounted is supposed to be a "when the physical count happened" and not "when the count was input into OB". A question for Manon.

Here's some new docs by Manon: https://pihemr.atlassian.net/wiki/spaces/OB/pages/3457777669/8.+Cycle+Count+-+MVP+Release#C.-Counted-by-%2F-Count-date

Reply:
let's leave it like that for now until we are sure about the expected behavior. I wouldn't want this ticket to block the work anymore.